express-app/routes/urls.json

```diff
@@ -6,7 +6,15 @@
 	"dashboard" : "/user/dashboard",
 	"getQuestions" : "/seccion/questions",
 	"api":  {
-	  "login" : "/perfiles-usuario/obtain-auth-token/",
-	  "questions" : "/captura/api-obtener-informacion-preguntas/"
-	}
+		"login" : "/perfiles-usuario/obtain-auth-token/",
+		"questions" : "/captura/api-obtener-informacion-preguntas/"
+	},
+	"member": "/estuido/member/",
+	"estudio": "/estudio/",
+	"estudioCreate": "/estudio/create/",
+	"estudioDelete": "/estudio/delete/:id",
+	"getEstudios" : "/estudio/get/",
+	"member": "/family/member/",
+	"memberCreate": "/family/member/create/"
```

Comment:
In the .hbs you send the studio id. So when you add a member the url doesn't match with this one so sends this message "Cannot POST /estudio/member/create/"